doc/design/registries/whitelist-registry.md

```diff
@@ -0,0 +1,151 @@
+# Whitelist Registries
+
+vSphere Integrated Containers 1.2 (VIC) added the ability to whitelist registry access in an installed VCH.  When a registry is added to the VCH at installed time, the VCH goes into 'whitelist' mode.  From this point on, the VCH will only allow access to registries in its list of whitelisted registries.  In this mode, users will not be able to access public registries, such as docker hub.
+
+## Specifying Whitelist Registries at Installation
+
+Whitelist registries can be declared during a VCH installation with *vic-machine* parameter, --whitelist-registry.  Two other vic-machine parameters affect whitelist registries, --registry-ca and --insecure-registry.  When whitelist registries are declared during installation, the latter two parameters acts as modifiers to the whitelist registries.
+
+Registry-ca declares additional certificates to verify access to registry servers secured with TLS.  If a registry is declared as a whitelist registry and not an insecure registry (discussed below), the VCH must have access to the server certificate to verify access.  The Photon-based VM that VIC uses has a base set of well-known certificates from public CAs.  If a whitelist registry uses a certificate that is not in that set of well-known certificates, the certificate must be uploaded to the VCH via vic-machine's --registry-ca parameter.
+
+Insecure-registry declares access to a registry server skip the TLS verification.  This modifies the whitelist label and takes precedence.  For instance, if a registry is declared with --whitelist-registry and with --insecure-registry, the VCH will assume the registry is an insecure whitelisted registry.  If the registry is listed with only --whitelist-registry, then the VCH will attempt to verify access using certificates.
+
+If a registry is declared with --insecure-registry but not with --whitelist-registry, vic-machine will add the insecure registries to the list of whitelist registries *IF* at least one whitelist registry was declared.
```

Comment:
Please also add description for vic-machine configure.
In general, if multiple configurations are related to each other, we'd tell user to update those configurations at one time, to avoid confusion. For example, if there is one white list registry added, and another one as insecure registry. If user removed the first white list registry, should we think white list registry is disabled?

In addition, we have listed reconfiguration for all current options, please add new issue to track the requirement for new configuration items. We already had a cut line for 1.2 release, so you might put the new issue into overall reconfiguration epic: #2911